cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/cache/SqlCache.kt

```diff
@@ -230,12 +208,13 @@ class SqlCache(
       return mutableListOf()
     }
 
+    val hashedIds = ids.asSequence().map { getIdHash(it) }.toSet()
```

Comment:
Same question here re: `toSet`-ing this when `getDataWith/WithoutRelationships` only requires a `Collection`?